pandas/tests/indexes/timedeltas/test_setops.py

```diff
@@ -73,3 +75,90 @@ def test_intersection_bug_1708(self):
         result = index_1 & index_2
         expected = timedelta_range('1 day 01:00:00', periods=3, freq='h')
         tm.assert_index_equal(result, expected)
+
+    @pytest.mark.parametrize("sort", [None, False])
+    def test_intersection_equal(self, sort):
```

Comment:
Similar to the `whatsnew` comments, reference an issue number as a comment beneath the function definition if possible.

Reply:
Added